docusaurus/docs/running-tests.md

```diff
@@ -259,8 +259,8 @@ script:
   - npm test
 ```
 
-1. Trigger your first build with a git push.
-1. [Customize your Travis CI Build](https://docs.travis-ci.com/user/customizing-the-build/) if needed.
```

Comment:
This is valid GitHub markdown so this commit was unnecessary.

Reply:
Not just GitHub flavored Markdown, but the Markdown spec itself.

> It’s important to note that the actual numbers you use to mark the list have no effect on the HTML output Markdown produces. The HTML Markdown produces from the above list is:

> If you instead wrote the list in Markdown like this:
> ```
> 1.  Bird
> 1.  McHale
> 1.  Parish
> ```
>
> you’d get the exact same HTML output.

Source: <https://daringfireball.net/projects/markdown/syntax#list>